testhelper/docker/resource/etcd/etcd.go

```diff
@@ -6,22 +6,49 @@ import (
 	"time"
 
 	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
 	etcd "go.etcd.io/etcd/client/v3"
 	"google.golang.org/grpc"
 
 	"github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource"
 )
 
 type Resource struct {
-	Client *etcd.Client
-	Hosts  []string
-	Port   int
+	Client           *etcd.Client
+	Hosts            []string
+	HostsSameNetwork []string
```

Comment:
possible to a small comment on what this variable will hold, also from name it feels like if this is some boolean variable.

Reply:
Yeah sure, will add a comment :+1: